packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -87,6 +88,20 @@ export class LayeredRuntimeResolverCreator {
         }
       }
 
+      // If delta marker is encountered, add resolver for `delta` query
+      if (parseMarker('delta', resolverElement.graphqlType.description)) {
+        const diffQuery = `get${resolverElement.graphqlType.name}Delta`
+
+        resolvers.Query[diffQuery] = (parent: any, args: any, context: any) => {
+          const filter: any = {};
+          if (args?.lastSync) {
+            filter.updatedAt = { gt: args.lastSync};
+          }
+          
+          return this.services[modelName].findBy(filter, undefined, undefined, context);
+        }
+      }
+
```

Comment:
Is there some other way to add the delta resolvers? This is kinda uncomfortable to do 😄

Reply:
This seems like a good approach to me, what makes it uncomfortable?

Reply:
I was a bit uncomfortable adding datasync stuff to the runtime itself, but nevermind that 😄

Reply:
Resolving this, feel free to open if I am mistaken